bin/deploy.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# This file is used to deploy Test Pilot to an S3 bucket.  It expects to be run
+# from the root of the Test Pilot directory and you'll need your S3 bucket name
+# in an environment variable $BUCKET
+
+# Questions?  Hit up #testpilot on IRC
+
+if [ ! -d "dist" ]; then
+    echo "Can't find /dist/ directory.  Are you running from the Test Pilot root?"
+    exit 1
+fi
+
+if [ -z "$BUCKET" ]; then
+    echo "The S3 bucket is not set. Failing."
+    exit 1
+fi
+
+
+# The basic strategy is to sync all the files that need special attention
+# first, and then sync everything else which will get defaults
+
+# This is not pretty, but until S3 supports arbitrary headers, here we are...
+
+# HTML; 10 minute cache
+aws s3 sync \
+  --cache-control "max-age=600" \
+  --content-type "text/html" \
+  --exclude "*" \
+  --include "*.html" \
+  --metadata '{"x-amz-meta-content-security-policy": "default-src '\''self'\''; connect-src '\''self'\'' analysis-output.telemetry.mozilla.org; font-src '\''self'\'' code.cdn.mozilla.net; script-src '\''self'\'' www.google-analytics.com; style-src '\''self'\'' code.cdn.mozilla.net; report-uri /__cspreport__;",
+               "x-amz-meta-strict-transport-security": "max-age=31536000; includeSubDomains; preload",
+               "x-amz-meta-x-content-type-options": "nosniff",
+               "x-amz-meta-x-frame-options": "SAMEORIGIN",
+               "x-amz-meta-x-xss-protection": "1; mode=block"}' \
+  --metadata-directive "REPLACE" \
+  --acl "public-read" \
+  dist/ s3://${BUCKET}/
+
+# JSON; 10 minute cache.
+aws s3 sync \
+  --cache-control "max-age=600" \
+  --content-type "application/json" \
+  --exclude "*" \
+  --include "*.json" \
+  --metadata '{"x-amz-meta-strict-transport-security": "max-age=31536000; includeSubDomains; preload"}' \
+  --metadata-directive "REPLACE" \
+  --acl "public-read" \
+  dist/ s3://${BUCKET}/
+
+# Everything else; cache forever, because it has hashes in the filenames
+aws s3 sync \
+  --delete \
```

Comment:
Does this need `--exclude "*.json" --exclude "*.html"` to avoid overwriting what the previous two commands did? Or will `s3 sync` skip the just-updated files?